src/webview/leetCodePreviewProvider.ts

```diff
@@ -11,19 +11,40 @@ class LeetCodePreviewProvider extends LeetCodeWebview {
 
     private node: IProblem;
     private description: IDescription;
+    private sideMode: boolean = false;
 
-    public async show(node: IProblem): Promise<void> {
+    public isSideMode(): boolean {
+        return this.sideMode;
+    }
+
+    public async show(node: IProblem, isSideMode: boolean = false): Promise<void> {
         this.description = this.parseDescription(await leetCodeExecutor.getDescription(node), node);
         this.node = node;
+        this.sideMode = isSideMode;
         this.showWebviewInternal();
+        if (this.sideMode) {
+            // Hide the side bar for better view area
+            commands.executeCommand("workbench.action.focusSideBar").then(() => {
+                commands.executeCommand("workbench.action.toggleSidebarVisibility");
+            });
+        }
     }
 
     protected getWebviewOption(): ILeetCodeWebviewOption {
-        return {
-            viewType: "leetcode.preview",
-            title: `${this.node.name}: Preview`,
-            viewColumn: ViewColumn.One,
-        };
+        if (!this.sideMode) {
+            return {
+                viewType: "leetcode.preview",
```

Comment:
Can we make the viewType as a member of the webview?

Reply:
Yes, we can make `viewType` and `iconPath` both inheritable members of webview.

Reply:
@Vigilans Will the change happen in 0.14.1?

Reply:
You mean add `iconPath` in 0.14.1?